src/agor/tools/hotkeys.py

```
@@ -278,6 +280,17 @@ def generate_meta_feedback_hotkey(
     except Exception as e:
         return f"❌ Failed to generate meta feedback: {e}"
 
+@lru_cache(maxsize=1)
+def _get_generate_meta_feedback_func():
+    """Helper function to lazy-load and cache the generate_meta_feedback function."""
+    try:
+        from agor.tools.agent_prompts import generate_meta_feedback
+        return generate_meta_feedback
+    except ImportError as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)